.github/labeler.yml

```
@@ -74,6 +74,10 @@ core/tracing:
 - changed-files:
   - any-glob-to-any-file: ['kong/tracing/**/*', 'kong/pdk/tracing.lua']
 
+core/wasm:
+- changed-files:
+  - any-glob-to-any-file: ['kong/runloop/wasm.lua', 'kong/runloop/wasm/**/*']
```

```suggestion
  - any-glob-to-any-file: ['kong/runloop/wasm.lua', 'kong/runloop/wasm/**']
```
Not sure we need that final /*

---

Yeah it looks odd to me too, but all of the other examples in the file use the same pattern